integreat_compass/static/src/index.ts

```diff
@@ -4,9 +4,12 @@ import "./js/display_filenames";
 import "./js/image_picker";
 import "./js/map";
 import { createIconsAt } from "./js/utils/create-icons";
+import { handleOfferDetails, countLengthOfReview, setLengthOfReview } from "./js/offer-details";
```

Comment:
Did you forget to push `./js/offer-details.js`?

Reply:
I wanted to do it in two PRs, but I have changed my mind now and will do it in one PR